docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -56,6 +56,9 @@ configuration identical on all cluster nodes.
 
      discovery.uri=https://<coordinator ip address>:<https port>
 
+   Note that using hostnames or fully qualified domain names for the URI is
```

Comment:
Does this limitation mean we should also restore the docs for the old way too?

Reply:
When we did the docs update we decided we do NOT want the old way anymore ...

Reply:
But as it currently stands there are some configurations which would still need the old way to be used.

Reply:
True ... but its also an edge case .. if your are worried about attacks inside the cluster and need TLS inside the cluster your security is already in trouble.. so its not a must have feature at all. And if you need that .. you can figure out the details.